config/accounts.go

```diff
@@ -156,6 +168,17 @@ func (a *AccountGDPR) PurposeVendorException(purpose consentconstants.Purpose, b
 	return found, true
 }
 
+// PurposeVendorExceptions returns the vendor exception map for a given purpose.
+func (a *AccountGDPR) PurposeVendorExceptions(purpose consentconstants.Purpose) (value map[openrtb_ext.BidderName]struct{}, exists bool) {
+	if a.PurposeConfigs[purpose] == nil {
+		return nil, false
+	}
+	if a.PurposeConfigs[purpose].VendorExceptionMap == nil {
+		return nil, false
+	}
+	return a.PurposeConfigs[purpose].VendorExceptionMap, true
+}
```

Comment:
Please see my earlier comment for re-using map responses. There could be 2 extra lookups that we don't to do in this code.